app/src/app/store/mapRenderSubs.ts

```diff
@@ -91,10 +95,38 @@ export const getRenderSubscriptions = (useMapStore: typeof _useMapStore) => {
       { equalityFn: shallowCompareArray },
     );
 
+  const _updateMapCursor = useMapStore.subscribe<MapStore["activeTool"]>(
```

Comment:
PP: This could also be in the map component and use tailwind `cursor-x` [properties](https://tailwindcss.com/docs/cursor)

Reply:
I like this suggestions but I'm sort of in a hurry so created new issue #145 so we can do this later